django_project/species/scripts/data_upload.py

```diff
@@ -212,11 +213,13 @@ def finish(self, headers):
                               "".format(self.created_list)
 
         if self.existed_list > 0 and self.created_list == 0:
-            success_message = "{} rows already exist in the database." \
+            success_message = "{} rows already exist and have been " \
+                              "overwritten in the database." \
```

Comment:
Let's remove the database, as it can sometimes be too intimidating for non-technical users. :)

Reply:
sure, I'll update the text and fix the conflicts too

Reply:
updated Dimas